zips/zip-0312.rst

```diff
@@ -153,50 +153,13 @@ using seed phrases to recover the original secret (which may be something
 desirable in the context of FROST).
 
 
-Re-randomizable FROST
----------------------
+Re-Randomized FROST
+-------------------
 
 To add re-randomization to FROST, follow the specification [#FROST]_ with the
 following modifications.
 
 
-Randomizer Generation
-'''''''''''''''''''''
-
-A new helper function is defined, which generates a randomizer. The
-`encode_signing_package` is defined as the byte serialization of the `msg`,
-`commitment_list` values as described in [#frost-serialization]_.
-Implementations MAY choose another encoding as long as all values (the message,
-and the identifier, binding nonce and hiding nonce for each participant) are
-unambiguously encoded.
-
-The function `random_bytes(n)` is defined in [#FROST]_ and it returns a buffer
-with `n` bytes sampled uniformly at random. The constant `Ns` is also specified
-in [#FROST]_ and is the size of a serialized scalar.
-
-::
-
-  randomizer_generate():
-
-  Inputs:
-  - msg, the message being signed in the current FROST signing run
-  - commitment_list = [(i, hiding_nonce_commitment_i,
-    binding_nonce_commitment_i), ...], a list of commitments issued by
-    each participant, where each element in the list indicates a
-    NonZeroScalar identifier i and two commitment Element values
-    (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list
-    MUST be sorted in ascending order by identifier.
-
-  Outputs: randomizer, a Scalar
-
-  def randomizer_generate(msg, commitment_list):
-    # Generate a random byte buffer with the size of a serialized scalar
-    rng_randomizer = random_bytes(Ns)
-    signing_package_enc = encode_signing_package(commitment_list, msg)
-    randomizer_input = rng_randomizer || signing_package_enc
-    return HR(randomizer_input)
-
-
 Round One - Commitment
 ''''''''''''''''''''''
 
```

Comment:
Typo just below here: "Roune One" -> "Round One"

Reply:
thanks, solved in 62833e6c